### ISSUE TITLE
owo-animals.json file and SQL database files severely out of date

### ISSUE THREAD

**Issue by ghost** [ChristopherBThai/Discord-OwO-Bot#295]
Trying to self-host the bot, which is apparently allowed according to the readme.
I found out the owo-animals.json file is over 3 years our of date, as such hunting does not work because many items such as cpatreon, patreon and gems are missing completely.
Is there any way to fetch an updated version of this file?

**Comment by gritor111** [comment]
cpatreon arent there but i think the wiki has a list of them, gems arent there but you can make it manually pretty easily since its only like 5 animals and same with patreons. It seems like you have most of the bot already done, are you doing this on windows? cus ive tried to do it and didnt succeed

**Comment by Togira123** [comment]
@ChristopherBThai update the json when

**Comment by gritor111** [comment]
that aint happening lmao

**Comment by gritor111** [comment]
Would you guys be interested in trying to try to self host the bot? fill in all the holes ect

**Comment by Togira123** [comment]
not really, we don't get any support whatsoever so my motivation isn't very high

**Comment by gritor111** [comment]
If you ever want to ill be happy to help
